jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -303,7 +304,7 @@
                                     (send self :get-proximity l/r :left))
                                  (send self :interpolatingp))
                        (unix::usleep 1000))))
-                 (send self :cancel-move-hand)
+                 (send self :cancel-move-hand l/r)
                  (send self :cancel-angle-vector)
```

Comment:
`(send self :cancel-angle-vector (send self :get-arm-controller arm))` doesn't work.
When I saw https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L646-L655, I found that I have to pass controller actions.
We can't assign controllers as `:rarm-controller` in current implementation.